code/_onclick/hud/plane_master.dm

```diff
@@ -24,12 +24,6 @@
 	blend_mode = BLEND_MULTIPLY
 	alpha = 255
 
-/atom/movable/screen/plane_master/openspace/backdrop(mob/mymob)
```

Comment:
Why this change?

Reply:
its redundant, this is already being done on init